### PR TITLE
Add initial text to DID Document section.

### PR DESCRIPTION
**Pull request by msporny**
Add initial content to the data model section on DID Documents.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec/pull/30.html" title="Last updated on Oct 10, 2019, 1:07 AM UTC (68cc190)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec/30/3df087e...68cc190.html" title="Last updated on Oct 10, 2019, 1:07 AM UTC (68cc190)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -517,8 +517,14 @@ <h1>
 
     <section>
       <h2>
-Document
+DID Document
       </h2>
+      <p>
+A DID Document is the resource that is associated with a
+<a>decentralized identifier</a>. The DID Document typically contains a
```
Comment (dlongley) [inline index.html]:
```suggestion
<a>decentralized identifier</a>. The DID Document typically expresses
```

File: index.html [hunk 2]
```diff
@@ -517,8 +517,14 @@ <h1>
...
+list of public keys and services that can be used to interact with an entity
```
Comment (dlongley) [inline index.html]:
```suggestion
verification methods (such as public keys) and services that can be used to interact with an entity
```

File: index.html [hunk 3]
```diff
@@ -508,13 +508,18 @@ <h1>
...
     <section>
       <h2>
-Document
+DID Document
       </h2>
+     <p>
+A DID Document is the resource that is associated with a
+<a>decentralized identifier</a>. The DID Document typically expresses
+verification methods (such as public keys) and services that can be used to interact with an entity
+that is associated with a DID.
```
Comment (ChristopherA) [inline index.html]: +1

**Comment by msporny** [comment]
>  to this (although I'm willing to bet that we're going to get calls to clarify what an "entity that is associated with a DID" is).

Attempted a fix in 07e47146f3637db14d38aa5845ec4acd6de444bb by referring to DID Controller.

**Comment by msporny** [comment]
Multiple reviews, attempted to address all suggestions/comments, no objections, merging.

**Pull request merged**